svelte-kit-scss/src/routes/+layout.server.ts

```diff
@@ -0,0 +1,12 @@
+import { ENV } from '$lib/constants/env';
```

Comment:
I think if there's an authenticated section can create layout in there
like routes/(authenticated)/layout.server.ts that fetches the data.